xija/component/heat.py

```diff
@@ -640,6 +654,21 @@ def dvals(self):
             else:
                 self.calc_earth_vis_from_taco(ephems, q_atts)
 
+            # This next bit optionally checks to see if the solar ephemeris
+            # was passed in, and if it was it computes the fraction of the
+            # Earth's surface that is illuminated by the Sun. 
+            solar_xyzs = [getattr(self, 'solarephem0_{}'.format(x))
+                          for x in ('x', 'y', 'z')]
+
+            if not None in solar_xyzs:
```

Comment:
Linters will probably say this should be `if None not in solar_xyzs:`. I'm try to get away from nitpicky comments, but here there is real ambiguity about precedence `not (None in solar_xyzs)` or `(not None) in solar_xyzs`. It is probably the former since the code is working, but this could be worth fixing.

Reply:
Fixed this